### PR TITLE
Delete API and cors

### PR DESCRIPTION
**Pull request by trauman413**
## Jira Ticket Number
Include a link to the ticket here
https://gaming-backlog.atlassian.net/browse/GB-43
https://gaming-backlog.atlassian.net/browse/GB-77

## Short description
Include a 1-3 sentence description of what is being done (doesn't need to be fancy)
Added necessary CORS requirements for Add to Library ticket on frontend and added a delete game from library API

## Type of Change
- [x] New feature
- [ ] Bug fix
- [ ] Documentation
- [ ] Other (please specify)

### CODE REVIEW

File: src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/services/LibraryService.kt [hunk 1]
```diff
@@ -59,14 +59,26 @@ class LibraryService (
     }
   }
 
+  fun deleteGameFromLibrary(libraryId: String, gameId: String) {
```
Comment (ksadoff) [inline src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/services/LibraryService.kt:62]: Do we have a separate ticket for adding tests for this?

**Review by ksadoff — COMMENTED** [review]
Looks good just one question about testing

File: src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/services/LibraryService.kt (hunk 1 above)
Comment (trauman413) [reply src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/services/LibraryService.kt:62]: good point, honestly completely forgot to test this! added new test, but I also realized I never populated the LibraryService tests file?? I'll make a ticket for that (and more edge cases will be needed there)

**Review by ksadoff — APPROVED** [review]
LGTM!
